internal/resource/http.go

```diff
@@ -206,6 +207,11 @@ func (c HttpClient) getReaderWithHeader(url string, header http.Header) (io.Read
 
 	req.Header.Set("User-Agent", "Ignition/"+version.Raw)
 
+	err = addSystemInfoHeaders(&req.Header)
+	if err != nil {
+		return nil, 0, nil, err
```

Comment:
I'm not sure what are the specifications behind this PR, but is this case really an hard-failure? We weren't sending those headers before, so I'm assuming we aren't making them mandatory now, thus we could just continue here.

/cc @ajeddeloh @crawford

Reply:
@lucab Your right. Crawford mentioned this to me as well.

Reply:
Fixed